Data/Storable/Endian.hs

```diff
@@ -37,6 +38,12 @@ import GHC.Int
 newtype LittleEndian a = LE a
 newtype BigEndian    a = BE a
 
+getLittleEndian :: LittleEndian a -> a
```

Comment:
How about: `newtype LittleEndian a = LE { getLittleEndian :: a }` and likewise for BE? Suppose then line 17 is unnecessary.